test/test-mdl.html

```diff
@@ -12,7 +12,11 @@
     <script src="/src/fragment-mdl.js" async></script>
   </head>
   <body>
+    <div id="whatever" class="mdl-fragment" config>
+      <div class="mdl-fragment__param" header cache-control="must-revalidate"></div>
+    </div>
```

Comment:
@Anomeon what about this piece of code?
Can you write please an example for this config?